cmd/skopeo/copy.go

```diff
@@ -17,6 +17,9 @@ func copyHandler(context *cli.Context) error {
 
 	policyContext, err := getPolicyContext(context)
 	if err != nil {
+		if os.IsNotExist(err) {
+			return fmt.Errorf("No system policy found. Install the default-policy.json from the source repo: %v", err)
```

Comment:
fair. I just pulled the cloned source and hit the error, and thought that any kind of information would be helpful.